tools/cli/domain_commands.go

```diff
@@ -298,26 +295,29 @@ func (d *domainCLIImpl) DeprecateDomain(c *cli.Context) {
 	})
 	if err != nil {
 		if _, ok := err.(*types.EntityNotExistsError); !ok {
-			ErrorAndExit("Operation DeprecateDomain failed.", err)
-		} else {
-			ErrorAndExit(fmt.Sprintf("Domain %s does not exist.", domainName), err)
+			return ErrorAndPrint("Operation DeprecateDomain failed.", err)
 		}
-	} else {
-		fmt.Printf("Domain %s successfully deprecated.\n", domainName)
+		return ErrorAndPrint(fmt.Sprintf("Domain %s does not exist.", domainName), err)
 	}
+	fmt.Printf("Domain %s successfully deprecated.\n", domainName)
+	return nil
 }
 
 // FailoverDomains is used for managed failover all domains with domain data IsManagedByCadence=true
-func (d *domainCLIImpl) FailoverDomains(c *cli.Context) {
+func (d *domainCLIImpl) FailoverDomains(c *cli.Context) error {
 	// ask user for confirmation
 	prompt("You are trying to failover all managed domains, continue? Y/N")
```

Comment:
while you're in here:
```suggestion
	prompt("You are trying to failover all managed domains, continue? [y/N]")
```
to show the default choice and be a bit more normal looking